kubernetes/data_source_kubernetes_service_test.go

```diff
@@ -76,8 +80,9 @@ func testAccKubernetesDataSourceServiceConfig_basic(name string) string {
   }
   spec {
     port {
-      port        = 8080
-      target_port = 80
+      port         = 8080
+      target_port  = 80
+      app_protocol = "http"
```

Comment:
Extended `TestAccKubernetesDataSourceService_basic` to verify `app_protocol` instead of creating a new test.